cloudinit/config/cc_growpart.py

```diff
@@ -161,9 +161,17 @@ def resize(self, diskdev, partnum, partdev):
 
 class ResizeGpart(object):
     def available(self):
-        if not util.which('gpart'):
-            return False
-        return True
+        myenv = os.environ.copy()
+        myenv['LANG'] = 'C'
+
+        try:
+            (_out, err) = util.subp(["gpart", "help"], env=myenv, rcs=[0, 1])
+            if re.search(r"gpart recover ", err, re.DOTALL):
```

Comment:
Maybe I'm missing something, but if we aren't using a `.` then I don't think `DOTALL` does anything (https://docs.python.org/3/library/re.html#re.DOTALL):

```suggestion
            if re.search(r"gpart recover ", err):
```